Updated ru_ru.lang

Hy60koshk opened the pull request:
ru_ru.lang updated to BetterPvP 20.27.3 / Minimap 20.27.2.1

Hy60koshk committed:
Updated ru_ru.json

ru_ru.json updated to match 1.16.4 (BetterPvP 20.27.3 / Minimap 20.27.2.1)

Hy60koshk committed:
Delete ru_ru.json

Hy60koshk committed:
Updated ru_ru.lang

ru_ru.lang updated to 1.16.4 (BetterPvP 20.27.3 / Minimap 20.27.2.1)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/thexaero/Better-PVP-Translations?pullRequest=6) <br/>All committers have signed the CLA.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_edit_waypoint=Изменить точку
+gui.xaero_armour_status_settings=Настройки «Состояния брони»
+gui.xaero_choose_a_preset=Рекомендуемые варианты
+gui.xaero_choose_an_interface=Выберите стиль
```
Тут должно быть именно "интерфейс".

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_OK=ОК
+gui.xaero_add_edit=Доб./Ред.
+gui.xaero_new_waypoint=Установить отметку
+gui.xaero_initial=исход
```
Означает конец слова, а должно значить начало?

Hy60koshk replied:
> gui.xaero_initial=исход

тут я затупил. должно быть "метка" или "инициалы"

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_miscellaneous=Прочее
+gui.xaero_edit_mode=Режим редактирования
+gui.xaero_instructions=Инструкция
+gui.xaero_not_ingame=Чтобы войти в режим редактирования, нужно быть в игре!
```
Вы это читаете в режиме редактирования, просто не можете его полностью использовать, так что старое - лучше. Можно добавить слово "полностью".

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_howto_deselect=Чтобы сбросить выделение, нажмите ПКМ.
+gui.xaero_howto_center=Чтобы выровнять элемент по центру, нажмите «C».
+gui.xaero_different_centered=Некоторые элементы при выравнивании по центру имеют другой вид.
+gui.xaero_howto_flip=Чтобы перевернуть элемент, нажмите «F».
```
Нужно уточнить, что горизонтально.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_better_pvp_settings=Настройки Better PVP
+gui.xaero_server_disabled=Возможно, сервер не позволяет использовать некоторые функции мода.
+gui.xaero_waypoints=Отметки на карте
+gui.xaero_box_potion_effects=Интерфейс с информацией о действующих на персонажа эффектах
```
Нужна точка в конце, как у остальных.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_box_potion_effects=Интерфейс с информацией о действующих на персонажа эффектах
+gui.xaero_box_armour_status=Интерфейс с информацией о состоянии брони и предмете в руках.
+gui.xaero_box_sprint=Бегите, зажав только одну кнопку (настраиваемая клавиша).
+gui.xaero_box_sneak=Позволяет красться переключением кнопки, без необходимости удерживать её (настраиваемая клавиша).
```
Старое гораздо понятнее, но можно сделать "включать и выключать режим крадения". И еще, "её" здесь не подходит, потому что тут говорится про иную привязку клавиш, которая обычно "shift", а не про ту, что переключает режим. Можно написать "удерживать обычную клавишу крадения".

Hy60koshk replied:
Слова "крадение" не существует в русском языке ;)
Про кастомную кнопку переформулирую.

thexaero replied:
А звучит как слово :D
Позволяет красться без необходимости...?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_box_armour_status=Интерфейс с информацией о состоянии брони и предмете в руках.
+gui.xaero_box_sprint=Бегите, зажав только одну кнопку (настраиваемая клавиша).
+gui.xaero_box_sneak=Позволяет красться переключением кнопки, без необходимости удерживать её (настраиваемая клавиша).
+gui.xaero_box_notifications=Оповещения о различных опасностях: низкое здоровье, голод, опасность взрыва, попадание стрелы.
```
Оповещение именно о том, что в Вас стреляют из лука, а не о том, что попали в цель.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_box_sneak=Позволяет красться переключением кнопки, без необходимости удерживать её (настраиваемая клавиша).
+gui.xaero_box_notifications=Оповещения о различных опасностях: низкое здоровье, голод, опасность взрыва, попадание стрелы.
+gui.xaero_box_xp=Полученный или потерянный опыт пролетает по экрану.
+gui.xaero_box_quick_use=Позволяет использовать предметы на панели быстрого доступа, не переключаясь между предметами, а просто нажав клавишу. Удерживая клавишу, вы можете продолжительно использовать предметы, как если бы вы зажали правую кнопку, например, чтобы есть пищу.
```
Совершенно неясно, что клавиш несколько. Для объяснения также лучше использовать предложения покороче, как было раньше. И оба примера, которые были в старом варианте строки, лучше оставить.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_preset_fancy=Стильный
+gui.xaero_preset_classic=Классический
+gui.xaero_preset_social=Общительный
+gui.xaero_potion_status=Активные эффекты
```
Старый вариант лучше описывает. Можно даже "Состояние активных эффектов" написать.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_potion_status=Активные эффекты
+gui.xaero_archery_status=Запас стрел
+gui.xaero_sprint=Однокнопочный бег
+gui.xaero_sneak=Переключ. скрытность
```
Лучше всё-таки оставить крадение.

thexaero replied:
Не нравится мне слово "скрытность".

thexaero replied:
Приседание?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_hunger_setting=Опевещение о голоде
+gui.xaero_hp_setting=Оповещ. о низком HP
+gui.xaero_explosion_setting=Оповещение о взрыве
+gui.xaero_being_shot_setting=Оповещ. об обстреле
```
Зачем эти сокращения? В последней версии мода места должно быть достаточно.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_enchants_color=Цвет зачарований
+gui.xaero_durability=Износ в:
+gui.xaero_hunger_setting=Опевещение о голоде
+gui.xaero_hp_setting=Оповещ. о низком HP
```
В другом месте Вы оставили "здоровье".

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_explosion_setting=Оповещение о взрыве
+gui.xaero_being_shot_setting=Оповещ. об обстреле
+gui.xaero_xp_setting=Визуализация опыта
+gui.xaero_custom_settings=Свои настройки
```
Не думаю, что тут подходит это слово.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_xp_setting=Визуализация опыта
+gui.xaero_custom_settings=Свои настройки
+gui.xaero_reset_defaults=Сбросить настроки
+gui.xaero_quick_use=Панель быстрого доступа
```
Звучит так, будто мод добавляет новую панель, но это не так. И "панель быстрого доступа" существует в игре изначально (hotbar). Мод эту панель делает еще быстрее. Верно было бы написать "Быстрая панель быстрого доступа", но звучит это плохо. Думаю именно поэтому раньше использовалось "панель действий" для перевода "hotbar". 
Можно попробовать "Улучшенный быстрый доступ".

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_tiny=крошечный
+gui.xaero_small=маленький
+gui.xaero_medium=средний
+gui.xaero_large=большой
```
В настройках игры опции пишутся с заглавной буквы, так что лучше и в моде так оставить.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_large=большой
+gui.xaero_pvp_sprint=Улучшенный бег
+gui.xaero_quick_eat=Быстрое поедание
+gui.xaero_toggle_sneak=Переключ. скрытность
```
Лучше всё-таки "крадение".

thexaero replied:
Не нравится мне слово "скрытность".

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_quick_use_6=Ячейка быстрого доступа 6
+gui.xaero_quick_use_7=Ячейка быстрого доступа 7
+gui.xaero_quick_use_8=Ячейка быстрого доступа 8
+gui.xaero_quick_use_9=Ячейка быстрого доступа 9
```
"Улучшенный быстрый доступ 9"?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_waypoint_teleport=Телепорт.
+gui.xaero_waypoints_key=Открыть меню отметок
+gui.xaero_display_coords=Координаты
+gui.xaero_disable_enable=Вкл/выкл
```
Нужно оставить старый вариант.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_entity_colours=Цвета сущностей
+gui.xaero_players=Игроки
+gui.xaero_mobs=Мирные мобы
+gui.xaero_hostile=Злые мобы
```
Думаю, лучше "враждебные".

Hy60koshk replied:
Не влезет

thexaero replied:
Вражд. мобы?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_hostile=Злые мобы
+gui.xaero_items=Предметы
+gui.xaero_other=Проч.сущности
+gui.xaero_hunger_low=Вы голодны
```
Не подходит для названия настройки. Старый вариант был лучше.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_to=В
+gui.xaero_used_seed=Ключ генератора:
+gui.xaero_show_distance=Расстояние до отметок
+gui.xaero_waypoints_scale=Размер отметок
```
"Размер внутриигровых отметок"

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_show_distance=Расстояние до отметок
+gui.xaero_waypoints_scale=Размер отметок
+gui.xaero_block_colours=Цвета блоков
+gui.xaero_accurate=Реальные
```
Старый вариант лучше.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_waypoints_scale=Размер отметок
+gui.xaero_block_colours=Цвета блоков
+gui.xaero_accurate=Реальные
+gui.xaero_vanilla=Упрощённые
```
Это цвета с обычных карт-предметов, поэтому "vanilla".

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_delete_set=Удал. набор
+gui.xaero_delete_set_message=Вы уверены, что хотите удалить текущий набор отметок
+gui.xaero_delete_set_message2=Набор будет удалён из текущего мира.
+gui.xaero_toggle_waypoints=Вкл/выкл отметки в мире
```
На мини-карте.

Hy60koshk replied:
точно?

>"gui.xaero_toggle_waypoints": "Toggle Ingame Waypoints"

>gui.xaero_waypoints_scale=Размер отметок
 thexaero 10 days ago Owner "Размер внутриигровых отметок"

thexaero replied:
Вы правы насчет gui.xaero_toggle_waypoints.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_entity_info=Инфо о сущностях
+gui.xaero_toggle_slime=Вкл/выкл подсветку чанков слизней
+gui.xaero_toggle_grid=Вкл/выкл сетку чанков
+gui.xaero_compass_over_wp=Компас над метками
```
Отметками?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_box_entity_info=Панель полезной информации о живых сущностях, на которых вы смотрите (включая игроков).
+gui.xaero_entity_info_settings=Настройки панели информации о живых сущностях
+gui.xaero_entity_info_stay=Задержка скрытия инфо
+gui.xaero_entity_info_distance=Радиус воздействия
```
Радиус работы?

thexaero replied:
Мне не очень нравится слово "воздействие", тк звучит будто мод влияет на сущностей.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_entity_info_settings=Настройки панели информации о живых сущностях
+gui.xaero_entity_info_stay=Задержка скрытия инфо
+gui.xaero_entity_info_distance=Радиус воздействия
+gui.xaero_entity_info_max_hearts=Размер шкалы HP
```
Старое верно.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_current_biome=Показывать текущий биом
+gui.xaero_auto_map_size=Авто
+gui.xaero_reset_message=Вы уверены, что хотите сбросить настройки к значениям по умолчанию?
+gui.xaero_reset_message2=После этого нельзя будет восстановить прошлые настройки.
```
Старое короче и ничем не хуже.

thexaero replied:
Еще добавлю, что это неверно, потому что восстановить прошлые настройки - можно, если постараться. 
"Это действие не отменить." будет лучше.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_arrow_scale=Размер стрелки
+gui.xaero_arrow_colour=Цвет стрелки
+gui.xaero_smooth_dots=Гладкие точки
+gui.xaero_cave_zoom=Масштаб в пещерах
```
Неверно. Эта настройка не задает конкретный масштаб в пещерах. Она умножает обычный масштаб. Раньше было только два варианта: ON/OFF, откуда и перевод. Думаю, он и сейчас работает лучше всего.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_cave_zoom=Масштаб в пещерах
+gui.xaero_lock_player_heads=Всегда показ. головы игроков
+gui.xaero_safe_mode_box=Безопасный режим можно использовать в том случае, если мод не работает в обычном режиме. В этом случае будет использоваться функционал мода, работающий целиком на Java (CPU), вместо того, который задействует OpenGL (GPU). Но возможности мода в этом режиме ограничены.
+gui.xaero_height_limit=Макс. высота сущностей
```
Неверно. Эта настройка не задает максимальную высоту для сущностей. Она задает максимальную разницу между высотой игрока и сущности. Так что старый перевод был вернее, но можно придумать что-нибудь новое, если сможете уместить.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_use_world_map=Использовать чанки карты мира
+gui.xaero_patron_capes=Patreon-плащи
+gui.xaero_entity_radar=Радар
+gui.xaero_entity_info_potion_effects=Эффекты зелий (оффлайн)
```
Думаю, лучше оставить старый перевод.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_entity_info_potion_effects=Эффекты зелий (оффлайн)
+gui.xaero_entity_info_potion_effects_scale=Размер эффектов зелий
+gui.xaero_terrain_depth=Эффект глубины
+gui.xaero_terrain_slopes=Затемн. склоны
```
Склоны не только затемняются.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_team=Команда
+gui.xaero_world_server=Мир/Сервер
+gui.xaero_subworld_dimension=Подмир/Измерение
+gui.xaero_switch_waypoint_set=Переключить набор отметок
```
Думаю, лучше будет "Сменить набор отметок".

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_waypoint_opacity_map=Непрозр. фона меток на карте
+gui.xaero_hide_world_names=Скрыть названия миров/IP
+gui.xaero_team_colours=Цвета команд
+gui.xaero_teleport_default_command=Команда для телепортации
```
"Команда для телепорт. по умолчанию". Не знаю, есть ли столько места, но это важная деталь.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_teleport_default_command=Команда для телепортации
+gui.xaero_open_slime=Открыть по кнопке
+gui.xaero_options=Опции
+gui.xaero_make_automatic=Сделать миром по умолч.
```
Не совсем верно звучит. Я бы написал "Сделать мир/сервер автомат."

Hy60koshk replied:
Тут не уверен. Разве не понятнее будет, если "автоматические" миры/измерения будут переведены как "по умолчанию"? В чём принципиальная особенность автоматических миров, чтобы называться именно "автоматическими"?

thexaero replied:
Выбрав из списка автоматический мир и подмир (после смены вручную), ты включаешь автоматическую смену подмира пометок при смене реального игрового подмира. Автоматический подмир постоянно меняется. Думаю, что так просто более интуитивно. 
На самом деле сам выбор мира является автоматическим, а не мир, но лучше не усложнять.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_options=Опции
+gui.xaero_make_automatic=Сделать миром по умолч.
+gui.xaero_make_automatic_msg1=Вы уверены?
+gui.xaero_make_automatic_msg2=Эта операция поменяет местами отметки в выбранном мире и в мире по умолчанию, чтобы настройки выбранного мира как бы стали настройками по умолчанию. Продолжаейте только в том случае, если хорошо знаете, что делаете.
```
"данные выбранного мира и автоматического мира, чтобы выбранный мир как бы стал автоматическим"

Тут опечатка: Продолжаейте

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_make_automatic=Сделать миром по умолч.
+gui.xaero_make_automatic_msg1=Вы уверены?
+gui.xaero_make_automatic_msg2=Эта операция поменяет местами отметки в выбранном мире и в мире по умолчанию, чтобы настройки выбранного мира как бы стали настройками по умолчанию. Продолжаейте только в том случае, если хорошо знаете, что делаете.
+gui.xaero_make_multi_automatic_msg2=Эта операция поменяет местами отметки в выбранном измерении и в измерении по умолчанию, чтобы настройки выбранного измерения как бы стали настройками по умолчанию. Продолжаейте только в том случае, если хорошо знаете, что делаете.
```
То же, что и в предыдущей строке, но в этом случае "подмир". "Измерение" не подходит.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_make_automatic_msg2=Эта операция поменяет местами отметки в выбранном мире и в мире по умолчанию, чтобы настройки выбранного мира как бы стали настройками по умолчанию. Продолжаейте только в том случае, если хорошо знаете, что делаете.
+gui.xaero_make_multi_automatic_msg2=Эта операция поменяет местами отметки в выбранном измерении и в измерении по умолчанию, чтобы настройки выбранного измерения как бы стали настройками по умолчанию. Продолжаейте только в том случае, если хорошо знаете, что делаете.
+gui.xaero_make_multi_automatic_msg1=Вы уверены?
+gui.xaero_make_multi_automatic=Сделать измерением по умолч.
```
"Сделать автомат. подмиром"

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_make_multi_automatic_msg2=Эта операция поменяет местами отметки в выбранном измерении и в измерении по умолчанию, чтобы настройки выбранного измерения как бы стали настройками по умолчанию. Продолжаейте только в том случае, если хорошо знаете, что делаете.
+gui.xaero_make_multi_automatic_msg1=Вы уверены?
+gui.xaero_make_multi_automatic=Сделать измерением по умолч.
+gui.xaero_delete_world=Удалить данные мира
```
мира/сервера

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_delete_multi_world_msg1=Вы уверены, что хотите удалить все данные об отметках для выбранного подмира или измерения?
+gui.xaero_delete_world_msg2=Данное действие нельзя будет откатить.
+gui.xaero_delete_multi_world_msg2=Данное действие нельзя будет откатить.
+gui.xaero_delete_multi_world=Удалить данные измерения
```
подмира

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_delete_world_msg2=Данное действие нельзя будет откатить.
+gui.xaero_delete_multi_world_msg2=Данное действие нельзя будет откатить.
+gui.xaero_delete_multi_world=Удалить данные измерения
+gui.xaero_always_show_distance=Всегда показ. расст. до меток
```
отметок?

Hy60koshk replied:
тут, к сожалению, не влезет, но думаю, и так понятно, в чём суть)

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_divide_all_by_8=Координаты подмира / 8
+gui.xaero_multiply_msg1=Эта опция нужна только для исправления некорректных координат отметок.
+gui.xaero_multiply_msg2=Вы уверены, что хотите УМНОЖИТЬ НА 8 ВСЕ координаты подмира?
+gui.xaero_divide_msg2=Вы уверены, что хотите РАЗДЕЛИТЬ НА 8 ВСЕ координаты подмира?
```
координаты отметок подмира?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_multiply_msg2=Вы уверены, что хотите УМНОЖИТЬ НА 8 ВСЕ координаты подмира?
+gui.xaero_divide_msg2=Вы уверены, что хотите РАЗДЕЛИТЬ НА 8 ВСЕ координаты подмира?
+gui.xaero_share=Поделиться
+gui.xaero_share_msg1=Вы уверены, что хотите поделиться координатами данной отметки §cСО ВСЕМИ§f в чате?
```
поделиться отметкой

Hy60koshk replied:
Не влезет, там очень маленькие кнопочки. Нельзя их, кстати, побольше сделать? Может, вообще сбоку расположить?
![image](https://user-images.githubusercontent.com/20334588/101258304-b638cb80-3732-11eb-8f13-f49113d2916f.png)

thexaero replied:
Этот текст не на кнопке, так что должно влезть :D Мне не понравилось "поделиться координатами данной отметки", тк не только координаты передаются. 
Когда кнопки внизу горизонтально, то меньше шанс попасть не туда (из-за ширины и высоты). Да и места сильно больше не станет, если сбоку. Желательно, чтобы на разрешении по умолчанию всё помещалось.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_divide_msg2=Вы уверены, что хотите РАЗДЕЛИТЬ НА 8 ВСЕ координаты подмира?
+gui.xaero_share=Поделиться
+gui.xaero_share_msg1=Вы уверены, что хотите поделиться координатами данной отметки §cСО ВСЕМИ§f в чате?
+gui.xaero_share_msg2=Удостоверьтесь, что тем самым вы не выдадите какое-либо секретное местоположение
```
Точка в конце?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_share=Поделиться
+gui.xaero_share_msg1=Вы уверены, что хотите поделиться координатами данной отметки §cСО ВСЕМИ§f в чате?
+gui.xaero_share_msg2=Удостоверьтесь, что тем самым вы не выдадите какое-либо секретное местоположение
+gui.xaero_roof=Крыша
```
Потолок?
Думаю, лучше подходит по смыслу.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_share_msg2=Удостоверьтесь, что тем самым вы не выдадите какое-либо секретное местоположение
+gui.xaero_roof=Крыша
+gui.xaero_partial=частично
+gui.xaero_full=целиком
```
Лучше, если опции с заглавной.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_show_light_level=Уровень освещения
+gui.xaero_up=[Подняться]
+gui.xaero_down=[Опуститься]
+gui.xaero_display_all_sets=Вкл/выкл отрисовку всех наборов отметок
```
Не думаю, что есть столько места.

Hy60koshk replied:
Странно, я почему-то не заметил такого. Эта опция есть в 1.16.3?
Поправил.

thexaero replied:
Это описание привязки клавиш. Есть в 1.16.3.  Может, там и было место, но я сомневался :S

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_display_all_sets=Вкл/выкл отрисовку всех наборов отметок
+gui.xaero_waypoint_name=Название отметки
+gui.xaero_set_name=Название набора
+gui.xaero_render_layer=Слоёв отрисовки оверлея
```
Слой отрисовки панелей?

Hy60koshk replied:
А в чём суть этой настройки?

thexaero replied:
В Forge есть несколько событий во время рисования игрового интерфейса, где моды могут вставит своё слово. Я назвал это слоями, и эта настройка позволяет выбрать слой. В Fabric она ничего не делает. 
Нужна она для того, чтобы бороться со случаями, где моды отменяют событие (слой), тем самым затыкая остальные моды, например миникарту. Происходит довольно часто.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_waypoint_name=Название отметки
+gui.xaero_set_name=Название набора
+gui.xaero_render_layer=Слоёв отрисовки оверлея
+gui.xaero_visit_needed=§eЧтобы конвертировать данное измерение, вам нужно сначала в нём побывать!
```
"Вам необходимо побывать в этом измерении, для того чтобы конвертировать данные в новый формат!"

Основной смысл этой строки в "Вам необходимо побывать в этом измерении", и на этом можно по сути закончить. Игрок не поставил себе целью "конвертировать формат", просто это необходимо для телепортации. Остальная часть строки объясняет, что при этом произойдет.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_render_layer=Слоёв отрисовки оверлея
+gui.xaero_visit_needed=§eЧтобы конвертировать данное измерение, вам нужно сначала в нём побывать!
+gui.xaero_mp_cross_tp=Телепорт между подмирами (по сети)
+gui.xaero_cross_tp=Телепорт между подмирами
```
Здесь уж точно имеется в виду "измерение".

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_mp_cross_tp=Телепорт между подмирами (по сети)
+gui.xaero_cross_tp=Телепорт между подмирами
+gui.xaero_unreachable_dimension=Недосягаемое измерение назначения
+gui.xaero_show_time=Отображать время
```
игровое время

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_12h=12ч
+gui.xaero_temporary=временная
+gui.xaero_restore=Восстановить
+gui.xaero_biomes_vanilla=Биомы в схематич. режиме
```
У Вас был следующий перевод:
gui.xaero_vanilla=Упрощённые
К сожалению, оба перевода не передают смысл, но это непросто.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_waypoint_distance_vertical_visibility_angle=Угол видимости меток (верт)
+gui.xaero_teleportation=Телепортация
+gui.xaero_teleportation_tooltip=Если выключить, то включить можно будет уже только через текстовый файл конфигурации!
+gui.xaero_use_multiworld_tooltip=Отключите, если находитесь на простом сервере с одним миром (без миров для лобби, отдельных режимов игры, мини-игр и прочего). Поиск доп. миров на таких серверах, особенно если на самом сервере не стоит данный мод, может привести к возникновению проблем.
```
Если на сервере стоит мод, то проблем не должно быть. Лучше перефразировать концовку.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_world_teleport_command=Команда для телепортации
+gui.xaero_use_multiworld=Поиск доп. миров
+gui.xaero_use_default=По умолчанию
+gui.xaero_centered_enlarged=На центр при развороте
+gui.xaero_zoomed_out_enlarged=Увелич. масштаб при развороте
```
Здесь должна быть отсылка к
gui.xaero_enlarge_map=Увеличить мини-карту

А также я бы перевёл "zoom out" как "уменьшить масштаб".

Hy60koshk replied:
> Здесь должна быть отсылка к gui.xaero_enlarge_map=Увеличить мини-карту

Она как бы есть... но да, надо что-то другое придумать вместо "разворота"

> А также я бы перевёл "zoom out" как "уменьшить масштаб".

Да, популярная ошибка - по ощущениям действительно хочется так сказать) Но на самом деле отдаление - это увеличение масштаба.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_sort_unsorted=выкл
+gui.xaero_sort_name=название
+gui.xaero_sort_symbol=метка
+gui.xaero_sort_distance=расстояние
```
С заглавной?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_sort_name=название
+gui.xaero_sort_symbol=метка
+gui.xaero_sort_distance=расстояние
+gui.xaero_box_cave_maps=Режим отрисовки карты, при котором отображается интерьер пещер или построек. Размер «крыши» - это размер квадрата (в блоках), который должен находиться над головой игрока (закрывать от него небо), чтобы произошло переключение в режим отображения пещер.
```
Тут тоже "потолка", если решите поменять.
Закрывание неба в оригинале не упоминается, так что лучше его убрать.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_sort_symbol=метка
+gui.xaero_sort_distance=расстояние
+gui.xaero_box_cave_maps=Режим отрисовки карты, при котором отображается интерьер пещер или построек. Размер «крыши» - это размер квадрата (в блоках), который должен находиться над головой игрока (закрывать от него небо), чтобы произошло переключение в режим отображения пещер.
+gui.xaero_box_entity_amount=Ограничение числа отображаемых сущностей для каждой категории сещностей
```
Опечатка в последнем слове и нет точки в конце.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_sort_distance=расстояние
+gui.xaero_box_cave_maps=Режим отрисовки карты, при котором отображается интерьер пещер или построек. Размер «крыши» - это размер квадрата (в блоках), который должен находиться над головой игрока (закрывать от него небо), чтобы произошло переключение в режим отображения пещер.
+gui.xaero_box_entity_amount=Ограничение числа отображаемых сущностей для каждой категории сещностей
+gui.xaero_box_distance=Для какиех отметок в мире отображать расстояние
```
каких* и в конце нужна точка.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_box_cave_maps=Режим отрисовки карты, при котором отображается интерьер пещер или построек. Размер «крыши» - это размер квадрата (в блоках), который должен находиться над головой игрока (закрывать от него небо), чтобы произошло переключение в режим отображения пещер.
+gui.xaero_box_entity_amount=Ограничение числа отображаемых сущностей для каждой категории сещностей
+gui.xaero_box_distance=Для какиех отметок в мире отображать расстояние
+gui.xaero_box_always_distance=Отображать расстояние до отметки, даже если она совсем рядом с игроком.
```
Нужно упомянуть
gui.xaero_show_distance=Расстояние до отметок
как в оригинале.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_box_distance=Для какиех отметок в мире отображать расстояние
+gui.xaero_box_always_distance=Отображать расстояние до отметки, даже если она совсем рядом с игроком.
+gui.xaero_box_arrow_scale=Размер стрелки игрока, используемой в невращающемся варианте карты, и в некоторых других случаях.
+gui.xaero_potion_effects_blink=Моргать при истечении
```
истекании? 
Сейчас звучит будто полностью истекло.

Hy60koshk replied:
Полностью истекло - было бы "по истечении".
А "истекание" - слово, конечно, существующее, в отличие от крадения, но тоже существующее только благодаря гибкости языка и на практике почти не применяемое) Да и по смыслу на самом деле почти не отличается.

М.б. "моргать на последних секундах" тогда?)

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_box_cave_zoom=Насколько приблизить карту, когда включается режим отображения пещер
+gui.xaero_box_entity_depth=Рисовать сущности более тёмными, если они не на одной высоте с вами
+gui.xaero_box_height_limit=Предельная разница в высоте (ось Y) между вами и сущностями, когда эти сущности перестают отображаться на карте
+gui.xaero_box_compass_over_wp=В случае их наложения друг на друга, обозначения сторон света (N, E, S, W) будут отрисованы поверх значков отметок на мини-карте .
```
Не хватает точек в конце.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_potion_effects_blink=Моргать при истечении
+gui.xaero_box_arrow_color=Цвет стрелки игрока, используемой в невращающемся варианте карты, и в некоторых других случаях.
+gui.xaero_box_cave_zoom=Насколько приблизить карту, когда включается режим отображения пещер
+gui.xaero_box_entity_depth=Рисовать сущности более тёмными, если они не на одной высоте с вами
```
Затемнять сущности в зависимости от разницы вашей высоты и высот сущностей?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_box_height_limit=Предельная разница в высоте (ось Y) между вами и сущностями, когда эти сущности перестают отображаться на карте
+gui.xaero_box_compass_over_wp=В случае их наложения друг на друга, обозначения сторон света (N, E, S, W) будут отрисованы поверх значков отметок на мини-карте .
+gui.xaero_toggle_map_waypoints=Вкл/выкл отметки на мини-карте
+gui.xaero_always_entity_nametags=Вседа показ. имена существ
```
Неверно. 
"Всегда показ. бирки сущностей"

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_minimap_text_align=Выравнивание текста
+gui.xaero_center=по центру
+gui.xaero_left=слева
+gui.xaero_right=справа
```
С заглавной.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_center=по центру
+gui.xaero_left=слева
+gui.xaero_right=справа
+gui.xaero_show_angles=Угол поворота игрока
```
Углы?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_left=слева
+gui.xaero_right=справа
+gui.xaero_show_angles=Угол поворота игрока
+gui.xaero_box_always_entity_nametags=Вседа отображать имена сущностей, имеющих именные жетоны, вне зависимости от настроек «Радара».
```
Бирки?

Hy60koshk replied:
Там обычные игровые Nametags имеются же в виду, которые для петов?

thexaero replied:
Да, и они переведены в игре как бирки, насколько я знаю.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_right=справа
+gui.xaero_show_angles=Угол поворота игрока
+gui.xaero_box_always_entity_nametags=Вседа отображать имена сущностей, имеющих именные жетоны, вне зависимости от настроек «Радара».
+gui.xaero_box_teleport_default_command=Команда, которая используется для телепортации к отметке, если не указана отдельная команда для применения на конкретном сервере (в меню «Точки на карте»).
```
Отметки на карте?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_arrow=стрелкой
+gui.xaero_main_entity_as=Обозначать игрока
+gui.xaero_dot=точкой
+gui.xaero_crosshair=крестом
```
С заглавной.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_waypoints_edit_hide=Скрыть
+gui.xaero_waypoints_edit_show=Показать
+gui.xaero_teleport_anyway=Всё равно телепортироваться
+gui.xaero_teleport_shows_coordinates=Вы увидите координаты
```
Восклицательный знак не помешал бы тут!

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_waypoints_edit_show=Показать
+gui.xaero_teleport_anyway=Всё равно телепортироваться
+gui.xaero_teleport_shows_coordinates=Вы увидите координаты
+gui.xaero_render_all_wp_sets=Отобр. все наборы отметок
```
Отображ. все наборы отметок?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_waypoints_local=Локал.
+gui.xaero_player_arrow_opacity=Непрозрачность стрелки
+gui.xaero_box_waypoints_distance=Предельное расстояние, на котором будут видны локальные отметки. Не распространяется на глобальные отметки.
+gui.xaero_connect_with_auto=Связать с подмиром
```
Добавить связь подмиров.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_player_arrow_opacity=Непрозрачность стрелки
+gui.xaero_box_waypoints_distance=Предельное расстояние, на котором будут видны локальные отметки. Не распространяется на глобальные отметки.
+gui.xaero_connect_with_auto=Связать с подмиром
+gui.xaero_disconnect_from_auto=Отвязать от подмира
```
Удалить связь подмиров.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_box_waypoints_distance=Предельное расстояние, на котором будут видны локальные отметки. Не распространяется на глобальные отметки.
+gui.xaero_connect_with_auto=Связать с подмиром
+gui.xaero_disconnect_from_auto=Отвязать от подмира
+gui.xaero_world_connection_tooltip=Связывает или отвязывает выбранные подмиры от текущего мира по умолчанию. Связывайте только подмиры из одного файла сохранения. Например, вы можете связать подмиры, если портал Незера объединяет их. Подмиры, связанные с текущим подмиром, всегда находятся вверху списка и отмечены звёздочкой (*).
```
Выбранный подмир от автоматического?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_connect_with_auto=Связать с подмиром
+gui.xaero_disconnect_from_auto=Отвязать от подмира
+gui.xaero_world_connection_tooltip=Связывает или отвязывает выбранные подмиры от текущего мира по умолчанию. Связывайте только подмиры из одного файла сохранения. Например, вы можете связать подмиры, если портал Незера объединяет их. Подмиры, связанные с текущим подмиром, всегда находятся вверху списка и отмечены звёздочкой (*).
+gui.xaero_connect_with_auto_msg=Вы уверены, что хотите связать выбранные подмиры?
```
Вы уверены, что хотите добавить следующую связь подмиров?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_disconnect_from_auto=Отвязать от подмира
+gui.xaero_world_connection_tooltip=Связывает или отвязывает выбранные подмиры от текущего мира по умолчанию. Связывайте только подмиры из одного файла сохранения. Например, вы можете связать подмиры, если портал Незера объединяет их. Подмиры, связанные с текущим подмиром, всегда находятся вверху списка и отмечены звёздочкой (*).
+gui.xaero_connect_with_auto_msg=Вы уверены, что хотите связать выбранные подмиры?
+gui.xaero_disconnect_from_auto_msg=Вы уверены, что хотите удалить связь между выбранными подмирами?
```
Вы уверены, что хотите удалить следующую связь подмиров?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_world_connection_tooltip=Связывает или отвязывает выбранные подмиры от текущего мира по умолчанию. Связывайте только подмиры из одного файла сохранения. Например, вы можете связать подмиры, если портал Незера объединяет их. Подмиры, связанные с текущим подмиром, всегда находятся вверху списка и отмечены звёздочкой (*).
+gui.xaero_connect_with_auto_msg=Вы уверены, что хотите связать выбранные подмиры?
+gui.xaero_disconnect_from_auto_msg=Вы уверены, что хотите удалить связь между выбранными подмирами?
+gui.xaero_teleport_not_connected=Вы пытаетесь телепортироваться в подмир, который не связан с вашим текущим подмиром по умолчанию. Если вы уверены, что этот подмир относится к тому же файлу сохранения, то можете связать эти подмиры, чтобы иметь возможность телепортироваться между ними. Если же вы свяжете подмиры, не относящиеся к одному сохранению, то при телепортации вы можете оказаться в неожид ...
```
с вашим автоматическим подмиром?

к тому же файлу сохранения или подсерверу

можете связать эти подмиры, используя «Отметки на карте» -> «Опции» -> «Добавить связь подмиров». Это позволит вам телепортироваться между ними.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_connect_with_auto_msg=Вы уверены, что хотите связать выбранные подмиры?
+gui.xaero_disconnect_from_auto_msg=Вы уверены, что хотите удалить связь между выбранными подмирами?
+gui.xaero_teleport_not_connected=Вы пытаетесь телепортироваться в подмир, который не связан с вашим текущим подмиром по умолчанию. Если вы уверены, что этот подмир относится к тому же файлу сохранения, то можете связать эти подмиры, чтобы иметь возможность телепортироваться между ними. Если же вы свяжете подмиры, не относящиеся к одному сохранению, то при телепортации вы можете оказаться в неожид ...
+gui.xaero_ignore_heightmaps=Игнор серверных карт высот
```
Не учит. данные высот сервера?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_error_loading_properties=Ошибка при загрузке серверных настроек мини-карты. Пожалуйста, повторите попытку.
+gui.xaero_light_block=блоков
+gui.xaero_light_sky=неба
+gui.xaero_light_both=всего
```
С заглавной.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_light_block=блоков
+gui.xaero_light_sky=неба
+gui.xaero_light_both=всего
+gui.xaero_waypoints_bottom=Новые метки внизу списка
```
в низ?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_light_both=всего
+gui.xaero_waypoints_bottom=Новые метки внизу списка
+gui.xaero_box_waypoints_bottom=Новые создаваемые отметки помещать в низ списка отметок набора.
+gui.xaero_armour_main_hand=Предмет в основной руке
```
ведущей руке?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_waypoints_bottom=Новые метки внизу списка
+gui.xaero_box_waypoints_bottom=Новые создаваемые отметки помещать в низ списка отметок набора.
+gui.xaero_armour_main_hand=Предмет в основной руке
+gui.xaero_armour_off_hand=Предмет в другой руке
```
неведущей руке?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_entity_radar_type=Тип сущности:
+gui.xaero_entity_radar_color=Цвет:
+gui.xaero_entity_radar_visiblity=Показывать:
+gui.xaero_entity_radar_names=Показ. имена:
```
Лучше просто "Имена"

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_minimap_shape_circle=Круглая
+gui.xaero_slopes_legacy=По-старому
+gui.xaero_slopes_default=Вкл
+gui.xaero_names_list=На TAB
```
Спис. игрок.

Hy60koshk replied:
Какая версия Forge последняя нормальная (позже 1.16.3), чтобы все моды работали?
Я сильно сомневаюсь, что даже в таком виде влезет на кнопку

thexaero replied:
Мне кажется, что влезет. В оригинале - "List Players". Не понял вопрос про Forge, если честно :D 36.0.0?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_entity_radar_visiblity=Показывать:
+gui.xaero_entity_radar_names=Показ. имена:
+gui.xaero_entity_radar_icons=Иконки:
+gui.xaero_box_entity_radar_names=Отображать имена сущностей под точками. \n \n Вариант «На TAB» означает, что имена будут отображаться тогда, когда зажата кнопка отображения списка игроков (по умолчанию - TAB).
```
«Список игроков»

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_ignore_heightmaps=Игнор серверных карт высот
+gui.xaero_box_ignore_heightmaps=Некоторые сервера устанавливают свои значения для карт высот, из-за чего карта мира может отрисовываться некорректно. Иногда такой эффект может даже возникнуть из-за некоторых модов в сольной игре. Данная настройка призвана исправить все эти проблемы, правда, чуточку пострадает производительность. Не включайте данную опцию без надобности. После переключения значени ...
+gui.xaero_error_loading_properties=Ошибка при загрузке серверных настроек мини-карты. Пожалуйста, повторите попытку.
+gui.xaero_light_block=блоков
+gui.xaero_light_sky=неба
```
Тут имеется в виду тип света. Освещаются всегда блоки. Можно сделать "Свет неба", "Свет блоков".

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_box_entity_radar_names=Отображать имена сущностей под точками. \n \n Вариант «На TAB» означает, что имена будут отображаться тогда, когда зажата кнопка отображения списка игроков (по умолчанию - TAB).
+gui.xaero_light_overlay_status=Оверлей освещения включён
+gui.xaero_light_overlay=Оверлей освещения
+gui.xaero_light_overlay_type=Вычислять освещённость
```
"Оверлей освещения" подходит лучше, тк есть опция "Выкл".

Hy60koshk replied:
???

thexaero replied:
Оригинал:
Light Overlay: Off / Block / Sky / All

"Вычислять освещённость: Выкл" звучит странно, и даже убрав Выкл. Эта настройка должна называться "Оверлей освещения". Строки есть две отдельные для разных кнопок для поддержки языков (на всякий случай). _gui.xaero_light_overlay_ - это не название настройки, а название экрана. _gui.xaero_light_overlay_type_ - название настройки.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_day=День
+gui.xaero_tamed=Питомцы
+gui.xaero_untamed_color=Цвет диких мобов
+gui.xaero_icons_list=На TAB
```
Спис. игрок.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_untamed_color=Цвет диких мобов
+gui.xaero_icons_list=На TAB
+gui.xaero_icons_always=Всегда
+gui.xaero_icons_off=Откл.
```
Без иконок

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_icons_list=На TAB
+gui.xaero_icons_always=Всегда
+gui.xaero_icons_off=Откл.
+gui.xaero_box_entity_radar_icons=Отображать иконки или головы существ вместо разноцветных точек. \n \n Вариант «На TAB» означает, что иконки будут отображаться тогда, когда зажата кнопка отображения списка игроков (по умолчанию - TAB).
```
сущностей?

«Список игроков»

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_icons_always=Всегда
+gui.xaero_icons_off=Откл.
+gui.xaero_box_entity_radar_icons=Отображать иконки или головы существ вместо разноцветных точек. \n \n Вариант «На TAB» означает, что иконки будут отображаться тогда, когда зажата кнопка отображения списка игроков (по умолчанию - TAB).
+gui.xaero_player_names=Имена под головами
+gui.xaero_mp_cross_disabled=Телепортация между измерениями в мультиплеере по умолчанию запрещена. Недочёты в алгоритме распознавания дополнительных миров могут привести к смерти вашего персонажа. Включить такую телепортацию можно в настройках мода. Но если баганёт, то потом не жалуйтесь!
+gui.xaero_box_lock_player_heads=Всегда отображать головы игроков вместо цветных точек.
+gui.xaero_entity_nametags=Имена существ
+gui.xaero_box_entity_nametags=Отображать имена сущностей, имеющих именные жетоны.
+gui.xaero_light_levels=Освещение
```
Этих строк уже не существует в оригинальной файле.

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,397 @@
+gui.xaero_entity_nametags=Имена существ
+gui.xaero_box_entity_nametags=Отображать имена сущностей, имеющих именные жетоны.
+gui.xaero_light_levels=Освещение
+gui.xaero_requires_reload=Чтобы изменения вступили в силу, требуется перезаход в мир!
+gui.xaero_flowers=Загружать цветы
+gui.xaero_render_arrow=Стрелка игрока
```
Эти строки из другого мода.

thexaero reviewed (commented):
Огромное Вам спасибо за проделанную работу! 
С очень многими исправлениями согласен полностью. Я и сам планировал переделать и улучшить перевод, тк текущий уже очень устарел и был в основном написан на скорую руку.
В Вашем переводе я также нашел довольно много вещей, которые хотел бы улучшить, исправить или обсудить. 
Я могу и сам всем этим заняться, если Вы так пожелаете. 
Еще раз, большое спасибо за помощь!

Hy60koshk commented:
Хорошо, спасибо за столь насыщенную обратную связь.
Через несколько дней — как вылезу с постельного режима — приду комментировать.
Если же через пару недель всё ещё не прорежусь — значит помер (что вряд ли со мной случится, если только не буквально), и тогда дальнейшие решения полностью за Вами =)

Hy60koshk committed:
Обновлён ru_ru.lang

Исправлена часть строк, убраны лишние строки, добавлены новые (про размер голов)

thexaero commented on ru_ru.lang:
```diff
@@ -5,48 +5,47 @@ gui.xaero_delete=Удалить
-gui.xaero_howto_preset=Чтобы выбрать из списка предзаготовленных вариантов настройки выбранного элемента, нажмите «Рекомендуемые варианты».
-gui.xaero_howto_save=Чтобы сохранить все изменения, нажмите «Подтвердить».
-gui.xaero_howto_cancel=Чтобы всё отменить, нажмите «Отменить».
+gui.xaero_howto_flip=Чтобы перевернуть элемент вверх ногами, нажмите "F".
```
Меняется местами право и лево. Почти зеркально, но я бы так не назвал. Горизонтально?

Hy60koshk replied:
Да, тут я что-то помутнение словил при прошлом редактировании

thexaero replied:
Ничего :)

thexaero commented on ru_ru.lang:
```diff
@@ -112,13 +111,15 @@ gui.xaero_quick_use_8=Ячейка быстрого доступа 8
+gui.xaero_waypoints_key=Точки на карте
 gui.xaero_display_coords=Координаты
-gui.xaero_disable_enable=Вкл/выкл
+gui.xaero_disable_enable=Выкл/вкл
```
Думаете, подходит в контексте скрывания пометок?

Hy60koshk replied:
Да, вполне. Не идеально, но учитывая размер кнопки, это нормальная практика

thexaero replied:
На кнопке будет только одно из двух слов, так что места больше, чем кажется.

thexaero commented on ru_ru.lang:
```diff
@@ -158,16 +159,16 @@ gui.xaero_from=Из
 gui.xaero_block_colours=Цвета блоков
-gui.xaero_accurate=Реальные
-gui.xaero_vanilla=Упрощённые
+gui.xaero_accurate=Настоящие
```
Чем плохо слово точные?

thexaero commented on ru_ru.lang:
```diff
@@ -197,34 +198,33 @@ gui.xaero_entity_info_numbers=Здоровье в цифрах
 gui.xaero_smooth_dots=Гладкие точки
-gui.xaero_cave_zoom=Масштаб в пещерах
-gui.xaero_lock_player_heads=Всегда показ. головы игроков
+gui.xaero_cave_zoom=Измен. масштаб в пещерах
```
масштаба?

thexaero commented on ru_ru.lang:
```diff
@@ -197,34 +198,33 @@ gui.xaero_entity_info_numbers=Здоровье в цифрах
+gui.xaero_cave_zoom=Измен. масштаб в пещерах
 gui.xaero_safe_mode_box=Безопасный режим можно использовать в том случае, если мод не работает в обычном режиме. В этом случае будет использоваться функционал мода, работающий целиком на Java (CPU), вместо того, который задействует OpenGL (GPU). Но возможности мода в этом режиме ограничены.
-gui.xaero_height_limit=Макс. высота сущностей
+gui.xaero_height_limit=Отобр.сущ. на расст. по высоте
```
Как-то запутанно!

thexaero commented on ru_ru.lang:
```diff
@@ -233,22 +233,20 @@ gui.xaero_switch_waypoint_set=Переключить набор отметок
 gui.xaero_make_automatic_msg1=Вы уверены?
-gui.xaero_make_automatic_msg2=Эта операция поменяет местами отметки в выбранном мире и в мире по умолчанию, чтобы настройки выбранного мира как бы стали настройками по умолчанию. Продолжаейте только в том случае, если хорошо знаете, что делаете.
-gui.xaero_make_multi_automatic_msg2=Эта операция поменяет местами отметки в выбранном измерении и в измерении по умолчанию, чтобы настройки выбранного измерения как бы стали настройками по умолчанию. Продолжаейте только в том случае, если хорошо знаете, что делаете.
+gui.xaero_make_automatic_msg2=Эта операция поменяет местами отметки в выбранном мире и в мире по умолчанию, чтобы настройки выбранного мира как бы стали настройками по умолчанию. Продолжайте только в том случае, если хорошо знаете, что делаете.
```
"данные выбранного мира и автоматического мира, чтобы выбранный мир как бы стал автоматическим"

Hy60koshk replied:
> Выбрав из списка автоматический мир и подмир (после смены вручную), ты включаешь автоматическую смену подмира пометок при смене реального игрового подмира. Автоматический подмир постоянно меняется. Думаю, что так просто более интуитивно.

Ничего не понял =)
У нас есть миры, есть подмиры, есть некое хранилище данных об отметках в них - для восприятия упростим до "файлов с отметками для каждого мира" (хотя я не знаю, как на самом деле это реализовано, но чисто чтобы условно разделить наборы точек для каждого подмира по отдельным "файлам"). Что именно с чем меняется местами, автовыбирается и т.д.?

thexaero replied:
Попробую объяснить на примере :D
Если выбраны _отметки_ "overworld (auto)", то, при переходе в самой игре в Незер, выбранные отметки сменятся на "the_nether (auto)". Если, находясь в Незере, ты сменяешь отметки на "overworld" (без auto), то автоматическая смена выключается. Перейдя теперь в the_end, выбранным останется "overworld". Выбрав "the_end (auto)", ты снова включаешь авто-смену.

thexaero commented on ru_ru.lang:
```diff
@@ -233,22 +233,20 @@ gui.xaero_switch_waypoint_set=Переключить набор отметок
+gui.xaero_make_multi_automatic=Сделать подмиром по умолч.
 gui.xaero_make_multi_automatic_msg1=Вы уверены?
-gui.xaero_make_multi_automatic=Сделать измерением по умолч.
+gui.xaero_make_multi_automatic_msg2=Эта операция поменяет местами отметки в выбранном подмире и в подмире по умолчанию, чтобы настройки выбранного подмира как бы стали настройками по умолчанию. Продолжайте только в том случае, если хорошо знаете, что делаете.
```
"данные выбранного подмира и автоматического подмира, чтобы выбранный подмир как бы стал автоматическим"

thexaero replied:
данные отметок*

thexaero commented on ru_ru.lang:
```diff
@@ -257,100 +255,100 @@ gui.xaero_item_tooltip_time=Время отображения
 gui.xaero_use_default=По умолчанию
 gui.xaero_centered_enlarged=На центр при развороте
-gui.xaero_zoomed_out_enlarged=Увелич. масштаб при развороте
+gui.xaero_zoomed_out_enlarged=Отдалить при развороте карты
```
> Она как бы есть... но да, надо что-то другое придумать вместо "разворота"

> Да, популярная ошибка - по ощущениям действительно хочется так сказать) Но на самом деле отдаление - это увеличение масштаба.

Я и подумал, что у Вас, скорее всего, верно, но Гугл мне тут же соврал :D 
На самом деле, мне логичнее, если 1:1000 > 1:1000000, как с дробями, поэтому и поверил Интернету.

Hy60koshk replied:
> На самом деле, мне логичнее, если 1:1000 > 1:1000000, как с дробями

Да, всё правильно. Это как раз проверочное действие, из которого следует, что я в прошлый раз перекрутил и ошибся. Прошу прощения. Но пускай всё равно "отдалить" будет, чтоб точно никто не запутался х)

thexaero replied:
Нет проблем! Согласен с "отдалить".

thexaero replied:
И всё-таки надо отослаться на строку gui.xaero_enlarge_map, которую Вы не перевели как "разворот".

thexaero commented on ru_ru.lang:
```diff
@@ -257,100 +255,100 @@ gui.xaero_item_tooltip_time=Время отображения
+gui.xaero_sort_name=Название
+gui.xaero_sort_symbol=Метка
+gui.xaero_sort_distance=Расстояние
+gui.xaero_box_cave_maps=Режим отрисовки карты, при котором отображается интерьер пещер или построек. Размер потолка - это размер квадрата (в блоках), который должен находиться над головой игрока (закрывать от него небо), чтобы произошло переключение в режим отображения пещер.
```
Лучше убрать часть о закрывании неба. Это не очень точно.

thexaero commented on ru_ru.lang:
```diff
@@ -257,100 +255,100 @@ gui.xaero_item_tooltip_time=Время отображения
+gui.xaero_box_cave_maps=Режим отрисовки карты, при котором отображается интерьер пещер или построек. Размер потолка - это размер квадрата (в блоках), который должен находиться над головой игрока (закрывать от него небо), чтобы произошло переключение в режим отображения пещер.
+gui.xaero_box_entity_amount=Ограничение числа отображаемых сущностей для каждой категории сущностей.
+gui.xaero_box_distance=Для каких отметок в мире отображать расстояние.
+gui.xaero_box_always_distance=Отображать расстояние до отметки, даже если отметка совсем рядом с игроком. Работает только при включённой настройке отображения расстояния до отметок.
```
Последнее предложение - неверно. Вот оригинал.
> This setting does not override the "Distance to WP" option.

Hy60koshk replied:
Ну правильно. Если не оверрайдит, значит если отключить "Distance to WP", то расстояние не будет вообще показываться, в т.ч. и данная опция не должна работать.

В русском, к сожалению, нет точного слова для перевода override - всё где-то в крайностях. Например, в данном случае подойдёт разве что "игнорирует". Из-за этого во многих случаях при общении с "простым пользователем" лучше вовсе избегать этого слова. Поэтому я не написал дословно.

thexaero replied:
Проблема у такого перевода в том, что настройка "Distance to WP" - не ВКЛ/ВЫКЛ. Если правильно помню, её и не выключить полностью.

thexaero replied:
Можно написать, что "Distance to WP" всё равно учитывается.

thexaero commented on ru_ru.lang:
```diff
@@ -257,100 +255,100 @@ gui.xaero_item_tooltip_time=Время отображения
 gui.xaero_restore=Восстановить
-gui.xaero_biomes_vanilla=Биомы в схематич. режиме
+gui.xaero_biomes_vanilla=Биомы в ванильном режиме
+gui.xaero_potion_effects_blink=Моргать при истечении
```
> Полностью истекло - было бы "по истечении".
> А "истекание" - слово, конечно, существующее, в отличие от крадения, но тоже существующее только благодаря гибкости языка и на практике почти не применяемое) Да и по смыслу на самом деле почти не отличается.
> 
> М.б. "моргать на последних секундах" тогда?)

Давай забудем про крадение :D Нужное слово.
Как докажешь, что "по истечении" и "при истечении" - разные вещи? Если поискать, оба варианта используются для обозначения "полностью истекло", в отличие от "при истекании". Для меня это инстинктивно, как делать и сделать, но я могу и ошибаться. 
Как насчет "моргать перед истечением"?

thexaero commented on ru_ru.lang:
```diff
@@ -257,100 +255,100 @@ gui.xaero_item_tooltip_time=Время отображения
-gui.xaero_box_height_limit=Предельная разница в высоте (ось Y) между вами и сущностями, когда эти сущности перестают отображаться на карте
-gui.xaero_box_compass_over_wp=В случае их наложения друг на друга, обозначения сторон света (N, E, S, W) будут отрисованы поверх значков отметок на мини-карте .
+gui.xaero_box_cave_zoom=Насколько приблизить карту, когда включается режим отображения пещер.
+gui.xaero_box_entity_depth=Затемнять точки сущностей в зависимости от разницы между высотой, на которой они расположены, и высотой, на которой стоите вы.
```
А что, если не стоим?

thexaero commented on ru_ru.lang:
```diff
@@ -257,100 +255,100 @@ gui.xaero_item_tooltip_time=Время отображения
+gui.xaero_box_compass_over_wp=В случае их наложения друг на друга, обозначения сторон света (N, E, S, W) будут отрисованы поверх значков отметок на мини-карте.
 gui.xaero_toggle_map_waypoints=Вкл/выкл отметки на мини-карте
-gui.xaero_always_entity_nametags=Вседа показ. имена существ
+gui.xaero_always_entity_nametags=Вседа показ. жетоны существ
```
Бирки, если брать официальный перевод за основу.

thexaero commented on ru_ru.lang:
```diff
@@ -257,100 +255,100 @@ gui.xaero_item_tooltip_time=Время отображения
-gui.xaero_disconnect_from_auto_msg=Вы уверены, что хотите удалить связь между выбранными подмирами?
+gui.xaero_connect_with_auto=Доб. связь подмиров
+gui.xaero_disconnect_from_auto=Убрать связь подмиров
+gui.xaero_world_connection_tooltip=Связывает или отвязывает выбранные подмиры от текущего мира по умолчанию. Связывайте только подмиры из одного файла сохранения. Например, вы можете связать подмиры, если портал Преисподней объединяет их. Подмиры, связанные с текущим подмиром, всегда находятся вверху списка и отмечены звёздочкой (*).
```
В официальном переводе Незер ведь.

thexaero replied:
И, чтобы не забылось:
> Выбранный подмир от автоматического?

thexaero commented on ru_ru.lang:
```diff
@@ -257,100 +255,100 @@ gui.xaero_item_tooltip_time=Время отображения
-gui.xaero_light_block=блоков
-gui.xaero_light_sky=неба
-gui.xaero_light_both=всего
+gui.xaero_light_block=Блоков
+gui.xaero_light_sky=Неба
+gui.xaero_light_both=Всего
```
> Тут имеется в виду тип света. Освещаются всегда блоки. Можно сделать "Свет неба", "Свет блоков".

thexaero replied:
Этот комментарий был больше связан с названием настройки, а не опций. Но, после изменения названия, нужно и тут подправить.

thexaero commented on ru_ru.lang:
```diff
@@ -368,7 +366,7 @@ gui.xaero_entity_radar_color=Цвет:
 gui.xaero_entity_radar_names=Показ. имена:
 gui.xaero_entity_radar_icons=Иконки:
-gui.xaero_box_entity_radar_names=Отображать имена сущностей под точками. \n \n Вариант «На TAB» означает, что имена будут отображаться тогда, когда зажата кнопка отображения списка игроков (по умолчанию - TAB).
+gui.xaero_box_entity_radar_names=Отображать имена сущностей под точками.\n \n Вариант «На TAB» означает, что имена будут отображаться тогда, когда зажата кнопка отображения списка игроков (по умолчанию - TAB).
```
«Список игроков»

thexaero commented on ru_ru.lang:
```diff
@@ -383,15 +381,6 @@ gui.xaero_untamed_color=Цвет диких мобов
-gui.xaero_on=Вкл
-gui.xaero_off=Выкл
\ No newline at end of file
+gui.xaero_box_entity_radar_icons=Отображать иконки или головы сущностей вместо разноцветных точек.\n \n Вариант «На TAB» означает, что иконки будут отображаться тогда, когда зажата кнопка отображения списка игроков (по умолчанию - TAB).
```
> точек.\n

Нужен пробер по техническим причинам.

«На TAB» >>> «Список игроков»

thexaero reviewed (commented):
Огромное спасибо за исправления!
Добавил новых комментариев. Извиняюсь, что так медлю!
С наступающим! Можете тоже не торопиться :)

Hy60koshk commented:
И вас с наступающим!
В общем, жду лекцию на тему автоматических миров)

Hy60koshk committed:
RU translation edits

thexaero commented on ru_ru.lang:
```diff
@@ -235,12 +235,12 @@ gui.xaero_waypoint_opacity_map=Непрозр. фона меток на карт
+gui.xaero_make_multi_automatic=Сделать авто-подмиром
 gui.xaero_make_multi_automatic_msg1=Вы уверены?
-gui.xaero_make_multi_automatic_msg2=Эта операция поменяет местами отметки в выбранном подмире и в подмире по умолчанию, чтобы настройки выбранного подмира как бы стали настройками по умолчанию. Продолжайте только в том случае, если хорошо знаете, что делаете.
+gui.xaero_make_multi_automatic_msg2=Эта операция поменяет местами отметки в выбранном подмире и в автоматическом, чтобы настройки выбранного подмира как бы стали настройками для автоматического подмира. Продолжайте только в том случае, если хорошо знаете, что делаете.
```
Перечитайте мои комментарии про эти строки . "Настройки" - не лучшее слово здесь.

thexaero commented on ru_ru.lang:
```diff
@@ -258,7 +258,7 @@ gui.xaero_multiply_msg1=Эта опция нужна только для исп
 gui.xaero_divide_msg2=Вы уверены, что хотите РАЗДЕЛИТЬ НА 8 ВСЕ координаты отметок подмира?
 gui.xaero_share=Поделиться
-gui.xaero_share_msg1=Вы уверены, что хотите поделиться координатами данной отметки §cСО ВСЕМИ§f в чате?
+gui.xaero_share_msg1=Вы уверены, что хотите поделиться информацией о данной отметке §cСО ВСЕМИ§f в чате?
```
Лучше просто сказать "поделиться данной отметкой". Вы пользовались этой функцией?

Hy60koshk replied:
Да, и при добавлении точки высвечивается информация о ней. Так что не вижу проблемы в слове "информация")
Но вообще, наверное, действительно будет лучше просто "отметкой", учитывая текст сообщения: "%игрок% поделился отметкой %имя_отметки% в %подмир%\n\n[Добавить]"
Кстати, да, само сообщение сейчас непереводимо. Подобные сообщения и "приветственное" сообщение об устаревшей версии можно как-нибудь сделать локализуемыми?
Да, и ещё лучше - добавить настройку для отключения проверки обновлений =)

thexaero replied:
Информация передается, да, но это звучит не так удобно, как есть на самом деле. Сделаю сообщение переводимым в следующем апдейте. Уже и строку добавил в оригинальный lang.
С переводом уведомления о новой версии - чуть сложнее, тк оно - не часть лишь Better PVP/миникарты, и для него в идеале нужны отдельные lang файлы. Скоро займусь и этим.
Уведомления можно отключить через config файл, но я добавил эту возможность по большей части для модпаков. Обычных пользователей нажатие кнопки _don't show for this update_ раз в неделю/обновление не укусит, если не хочется обновлять так часто. Иногда всё-таки стоит обновлять для своего же блага. Авторы модпаков знают это. А вот обычные пользователи, к сожалению, этого не делают, если не напоминать.

thexaero commented on ru_ru.lang:
```diff
@@ -280,7 +280,7 @@ gui.xaero_12h=12ч
 gui.xaero_restore=Восстановить
 gui.xaero_biomes_vanilla=Биомы в ванильном режиме
-gui.xaero_potion_effects_blink=Моргать при истечении
+gui.xaero_potion_effects_blink=Моргать перед истечением
```
Может "мигать"?

thexaero reviewed (commented):
Огромное спасибо! 
Я позже пройду по всем диалогам и закрою те, что уже (реально) устарели, чтобы вы обратили внимание на те, что еще важны.

Hy60koshk committed:
ru_ru(4)_final_final.lang

thexaero commented:
Спасибо!

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,386 @@
+gui.xaero_smooth_dots=Гладкие точки
+gui.xaero_cave_zoom=Измен. масштаб в пещерах
+gui.xaero_safe_mode_box=Безопасный режим можно использовать в том случае, если мод не работает в обычном режиме. В этом случае будет использоваться функционал мода, работающий целиком на Java (CPU), вместо того, который задействует OpenGL (GPU). Но возможности мода в этом режиме ограничены.
+gui.xaero_height_limit=Расст. до сущностей по высоте
```
Фильтр сущностей по высоте?

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,386 @@
+gui.xaero_options=Опции
+gui.xaero_make_automatic=Сделать авто-миром
+gui.xaero_make_automatic_msg1=Вы уверены?
+gui.xaero_make_automatic_msg2=Эта операция поменяет местами отметки в выбранном мире и в автоматическом мире, чтобы выбранный мир как бы стал автоматическим. Продолжайте только в том случае, если хорошо знаете, что делаете.
```
данные отметок*

thexaero commented on ru_ru.lang:
```diff
@@ -1,238 +1,386 @@
+gui.xaero_box_waypoints_distance=Предельное расстояние, на котором будут видны локальные отметки. Не распространяется на глобальные отметки.
+gui.xaero_connect_with_auto=Доб. связь подмиров
+gui.xaero_disconnect_from_auto=Убрать связь подмиров
+gui.xaero_world_connection_tooltip=Связывает или отвязывает выбранные подмиры от текущего автоматического мира. Связывайте только подмиры из одного файла сохранения. Например, вы можете связать подмиры, если портал Незера объединяет их. Подмиры, связанные с текущим подмиром, всегда находятся вверху списка и отмечены звёздочкой (*).
```
"немодифицированный портал Незера"?

thexaero commented:
Я закрыл все завершенные диалоги. Пожалуйста, перечитайте те, что я оставил. Там совсем немного. Лучше - по отдельным коммитам пройти, чтобы ничего не упустить. Уже выглядит очень неплохо! Огромное спасибо!

thexaero commented:
Огромное спасибо за Вашу работу!
Дальше я сам продолжу и добавлю обновленный перевод в следующем апдейте.
